Gems/Atom/Tools/AtomToolsFramework/Code/Source/Viewport/RenderViewportWidget.cpp

```diff
@@ -192,6 +203,28 @@ namespace AtomToolsFramework
     {
         m_time = time;
         m_controllerList->UpdateViewport({GetId(), AzFramework::FloatSeconds(deltaTime), m_time});
+
+        if (m_viewportContext)
+        {
+            float fpsLimit = 0.f;
+
+            if (auto console = AZ::Interface<AZ::IConsole>::Get())
+            {
+                console->GetCvarValue<float>("r_fps_limit", fpsLimit);
```

Comment:
suggestion: With this you might just be able to extern `r_fps_limit` and then read from it like a normal variable, instead of needing to call the `AZ::Interface<AZ::IConsole>...`